WC_Layers/OpenMC_PhotonTransport.py

```diff
@@ -0,0 +1,78 @@
+import openmc
+import numpy as np
+from Source_Mesh_Reader import extract_source_data, Files
+
+# Photon source densities from ALARA output 
+esd = extract_source_data(Files)
+
+def make_source(cells, mesh_file):
+      '''
+    Creates a list of OpenMC sources, complete with the relevant space and energy distributions
+    
+    inputs:
+        cells: list of OpenMC Cell objects
+        mesh_file: .h5/.h5m mesh onto which photon source will be distributed
+        
+    output:
+        source_list: list of OpenMC independent sources
+        total_mesh: OpenMC Unstructured Mesh object
+    '''
+    source_list = []
+    total_mesh = openmc.UnstructuredMesh(mesh_file, library='moab')
+    for index, (lower_bound, upper_bound) in enumerate(zip(bounds[:-1],bounds[1:])):
+        mesh_dist = openmc.stats.MeshSpatial(total_mesh, strengths=esd[mesh_index][:,index], volume_normalized=False)
+        energy_dist = openmc.stats.Uniform(a=lower_bound, b=upper_bound)
+        source_list.append(openmc.IndependentSource(space=mesh_dist, energy=energy_dist, strength=np.sum(esd[mesh_index][:, index]), particle='photon', domains=cells))
+    return source_list, total_mesh
+
+def tallies(total_mesh, tallied_cells):
+      '''
+    Creates tallies and assigns energy, spatial, and particle filters.
+    
+    inputs: 
+        total_mesh: OpenMC unstructured mesh object
+        tallied_cells: OpenMC Cell/iterable of OpenMC Cell objects/iterable of Cell ID #
+        
+    outputs:
+        talls: OpenMC Tallies object
+    '''
+    particle_filter = openmc.ParticleFilter('photon')

```

Comment:
You define this particle filter, which makes sense for a photon transport problem, but all the rest of your tally definition seems like it is meant for neutrons, especially the choice of scores, energy filter

Reply:
Yes, tally 1 originated from the neutron transport model. Perhaps a heating/absorption photon tally (with the appropriate variable names) would provide more relevant information?

Reply:
For R2S problems, it's customary to do an equivalent dose tally.